charts/cloudinfo/values.yaml

```diff
@@ -26,6 +26,7 @@ frontend:
 
   ingress:
     enabled: false
+    ingressClassName: Prefix
```

Comment:
**Question**: Are you sure you wanted to paste this here? I thought this is going to be the name of some default ingress class?

Reply:
Just noticed it myself and already fixed 🙂